include/eosio/vm/bitcode_writer.hpp

```diff
@@ -299,7 +299,12 @@ namespace eosio { namespace vm {
          body.code = fb.raw();
          body.size = op_index + 1;
          _base_offset += body.size;
+         op_index++;
```

Comment:
Might be clearer to move this to the first line of finalize and then use the `op_index`  value after instead of `op_index + 1`